python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -1031,55 +1017,23 @@ def _getIOTimeOut(self):
             _timeOut = 5. # 5sec hardcoded default
         return _timeOut
 
-    def _getWriteAccess(self):
-        """ tries to get write access to the object.
-        Raise LockingError (or so) on fail """
-        root = self._getRoot()
-        reg = root._getRegistry()
+    def _getSessionLock(self):
```

Comment:
I think it's safer that we don't release it until shutdown for now as this is a bit dangerous I feel.
I didn't think we could load an object if getting the lock failed, I assumed that was the point, or whether we were loading it but unable to use it in which case I'm even more worried about the way it used to be.

In this PR are we currently locking on loading in this case then?